Resources/Prototypes/DeltaV/Entities/Mobs/Customization/Markings/felinid.yml

```
@@ -3,7 +3,7 @@
   bodyPart: Tail
   markingCategory: Tail
   speciesRestriction: [Felinid, Human]
-  sponsorOnly: true
+  sponsorOnly: true # LPP-Sponsors
```

yamllint

[warning] too few spaces before comment

(comments)